src/redux/auth/auth-operations.ts

```diff
@@ -1,3 +1,4 @@
+import { IUser, IRefreshToken, IAuthInitialState } from './../types/typesStore';
```

Comment:
I think the next task can be about replace all relative paths to absolute (not manually, of course)

Reply:
I don't mind))

Reply:
@VladChornyi what do you think about automatically replace all paths to absolute? Maybe is there some tool?

Reply:
@kstroevsky I'm sure that it exists, but i didn't do that earlier and should find something